### ISSUE TITLE
Ask for WinARM64 Release

### ISSUE THREAD

**Issue by JeffersonQin** [ppy/osu#12210]
As ARM has become an increasingly popular architecture, I consider that the release for WinARM64 is also important. Machines like Surface Pro X needs this binary.

**Comment by smoogipoo** [comment]
This probably highly depends on Un4seen BASS supporting WinARM64. But doesn't it run via x86/64 emulation anyway?

**Comment by JeffersonQin** [comment]
Indeed, now microsoft only support the emulation of x86, and the x64 emulation is not supported yet.

**Comment by JeffersonQin** [comment]
> This probably highly depends on Un4seen BASS supporting WinARM64. But doesn't it run via x86/64 emulation anyway?

Also, the emulation can cause severe loss of performance, which is fatal for music games.

**Comment by sr229** [comment]
.NET 6.0 should allow ARM binaries, in which case, the only blocker is BASS, which I hope will gain support.

**Comment by JeffersonQin** [comment]
> .NET 6.0 should allow ARM binaries, in which case, the only blocker is BASS, which I hope will gain support.

Well, I found that BASS has already had the support for ARM64: http://www.un4seen.com/forum/?topic=16665.0

![image](https://user-images.githubusercontent.com/30655701/112836340-0a463200-90cd-11eb-8a05-8c05320698e1.png)

And also, as far as I know, .NET 5 has already supported the WinARM64 architecture. Anyway, thanks for replying.

**Comment by smoogipoo** [comment]
Ah interesting, I completely glossed over the UWP link because I didn't think it'd have ARM binaries in there. Seems like something that can be done then.

**Comment by sr229** [comment]
> > .NET 6.0 should allow ARM binaries, in which case, the only blocker is BASS, which I hope will gain support.
> 
> Well, I found that BASS has already had the support for ARM64: http://www.un4seen.com/forum/?topic=16665.0
> 
> ![image](https://user-images.githubusercontent.com/30655701/112836340-0a463200-90cd-11eb-8a05-8c05320698e1.png)
> 
> And also, as far as I know, .NET 5 has already supported the WinARM64 architecture. Anyway, thanks for replying.

.NET 5.0 and below only supports x86 on another platform. [.NET 6.0](https://devblogs.microsoft.com/dotnet/announcing-net-6-preview-1/) introduces native ARM64 support beyond Windows as this is the one that ships with Apple M1 Silicon support.

**Comment by smoogipoo** [comment]
@sr229 This doesn’t relate to macOS ARM. The two are different binaries. Fact of the matter is, you can publish with the `win-arm64` moniker in .NET 5 already, so support can be added at any point. It just needs someone to do it.

I’m locking this thread because this doesn’t need discussion.
